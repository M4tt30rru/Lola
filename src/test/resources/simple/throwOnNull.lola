##Import "std.lola"
##Find ##Expression(_exp) !! ##Identifier(_exc);
 ##replace if (##(_exp) == null) throw new ##(_exc)(); else ;

f() !! BadTimeException;
