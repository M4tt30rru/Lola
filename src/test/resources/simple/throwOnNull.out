if (f() == null)
 throw new BadTimeException();
else
 ;